digitaltwin_client/samples/digitaltwin_sample_sdk_info/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+#Copyright (c) Microsoft. All rights reserved.
+#Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+cmake_minimum_required(VERSION 2.8.11)
+#this is CMakeLists for digitaltwin_sample_sdk_info
+
+if(POLICY CMP0042)
+    cmake_policy(SET CMP0042 NEW)
+endif()
+
+compileAsC99()
+
+set(digitaltwin_sample_sdk_info_c_files
+    ./digitaltwin_sample_sdk_info.c
+)
+
+set(digitaltwin_sample_sdk_info_h_files
+    ./digitaltwin_sample_sdk_info.h
+)
+
+set(DIGITALTWIN_SAMPLE_SDK_INFO_INC_FOLDER ${CMAKE_CURRENT_LIST_DIR} CACHE INTERNAL "this is what needs to be included if using digitaltwin_client lib" FORCE)
```

Comment:
>"this is what needs to be included if using digitaltwin_client lib" [](start = 84, length = 67)

The "this is what..." comment I think we should remove from here and various other sample interfaces it was pre-existing.

I don't think as-is it adds much clarity and not sure if we need explicit in-line doc in 1st place.

Reply:
Should we remove "CACHE INTERNAL "this is what needs to be included if using digitaltwin_client lib" FORCE" completely so that the variable is not cached?  The comment is required if we want to cache the variable.

Reply:
Reading the cmake docs on [set](https://cmake.org/cmake/help/v3.0/command/set.html), I don't think we need the CACHE stuff here.  See `variable types in CMake` for details.

@danewalton - you've spent more time with cmake than I have.  Can you see any reason we'd want this?

Reply:
to promote best CMake practices this should not be included. if you wish to use an installed version of a lib there are other more modern ways to do it. [Example](https://github.com/Azure/azure-macro-utils-c/blob/5926caf4e42e98e730e6d03395788205649a3ada/CMakeLists.txt#L32) that I pushed to `macro-utils` which differentiates between the install directory and build directory.